tablecodec/tablecodec.go

```diff
@@ -209,6 +209,15 @@ func EncodeMetaKey(key []byte, field []byte) kv.Key {
 	return ek
 }
 
+// EncodeMetaKeyPrefix encodes the key prefix into meta key
+func EncodeMetaKeyPrefix(key []byte) kv.Key {
```

Comment:
I think you could just use `EncodeMetaKey(key, nil)`?

Reply:
In `EncodeBytes`, `nil` will be transfer to `[0, 0, 0, 0, 0, 0, 0, 0, 247]`